doc/dune-files.rst

```diff
@@ -1919,7 +1919,7 @@ The loading of the plugin is done using the facilities generated by
 generate_sites_module (Since 2.8)
 ---------------------------------
 
-Dune proposes some facilities for dealing with :ref:`sites` in a program. The
+Dune proposes some facilities for dealing with :ref:`sites<sites>` in a program. The
 ``generate_sites_module`` stanza will generate code for looking up the correct locations
 of the sites directories and for loading plugins. It works after installation
```

Comment:
```suggestion
of the sites' directories and for loading plugins. It works after installation
```

If this is plural "sites," it will need an apostrophe after the "s," as shown. If it's singular, it should appear like this: site's.